modules/ROOT/pages/_partials/sync/server-offline-and-conflict.adoc

```diff
@@ -188,17 +186,30 @@ const { conflictHandler, strategies } = require('@aerogear/voyager-conflicts')
 
 When there is no conflict, `conflictHandler.nextState(clientData)` is called and the data is persisted. When a conflict occurs, the following happens.
 
-* `conflictHandler.resolveOnServer` is called with the `serverWins` strategy. In this case, the `resolvedState` will be the current server record with an updated version number. *The resolvedState must be persisted as the version is updated.*
+* `conflictHandler.resolveOnServer` is called with the `clientWins` strategy. In this case, the `resolvedState` will be the new data provided by the client. The newly `resolvedState` should be persisted.
```

Comment:
can we avoid 'should'?
How about 'You need to persist the `resolvedState` data.' or similar?

Reply:
@darahayes  merged as i posted review, just wondering tho, is my understanding correct in the comment I made?

Reply:
@finp apologies for that. Your understanding is definitely correct. How about I just switch the word `should` with `must`. **The newly `resolvedState` must be persisted**. WDYT?